compiler/noirc_frontend/src/lexer/lexer.rs

```diff
@@ -541,6 +541,20 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_attribute_with_speed_marks() {
+        let input = r#"#[test(should_fail_with = "this can't compile")]"#;
```

Comment:
Can you update this text? It's a bit confusing to read "this can't compile" on a test that is expected to compile